Selected Questions/lec8.md

```diff
@@ -0,0 +1,33 @@
+# Lec 8- Solving Ax = b: Row Reduced Form R
+
+1.Find the value of c that makes it possible to solve Ax = b, and solve it: <br>
+        u + v + 2w = 2 <br>
+       2u + 3v - w = 5 <br>
+       3u + 4v + w = c: <br>
+       
+ <details>
+<summary>
+Answer
+</summary>
+c == 7 allows u = 1, v == 1, w == 0. The column space is a plane.
+</details>
+
+2.What conditions on b1, b2, b3, b4 make each system solvable? Solve for x:
+
+![Question2](Images/lec8_q2.jpg)
+
+<details>
+<summary>
+Answer
+</summary>
+Solvable if b2 = 2b1 and 3b1 - ,3b3 + b4 = 0. Then X = 
```

Comment:
`3b1 - ,` 
is there something missing here?

Reply:
The comma was unnecessary.